portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMPermissions.testcase

```diff
@@ -715,15 +715,15 @@ definition {
 			layoutName = "Documents and Media Page",
 			widgetName = "Documents and Media");
 
-		Navigator.gotoPage(pageName = "Documents and Media Page");
-
-		DMDocument.enableActionsMenuOnPortlet();
-
 		JSONFolder.addFolder(
 			dmFolderDescription = "DM Folder Description",
 			dmFolderName = "DM Folder Name",
 			groupName = "Guest");
 
+		Navigator.gotoPage(pageName = "Documents and Media Page");
```

Comment:
@DebSouza sometimes a few extra seconds are needed for the JSON request to process so the asset is not visible yet when the test tries to find it. Reordering these steps gives more time for that request to process without changing any essential flow so the test will be less flaky.

@Jeremy-Ch Please explain next time and also address this during the test design phase (you passed it in the steps listed [here](https://issues.liferay.com/browse/LPS-136833))